Modules/Devices/src/Devices.Infrastructure/PushNotifications/Connectors/Apns/ApplePushNotificationServiceConnector.cs

```diff
@@ -110,6 +110,6 @@ internal static partial class ApplePushNotificationServiceConnectorLogs
         EventId = 770700,
         EventName = "ApplePushNotificationServiceConnector.Sending",
         Level = LogLevel.Debug,
-        Message = "Sending push notification (type '{eventName}') to '{address}' with handle '{handle}'.")]
-    public static partial void Sending(this ILogger logger, string eventName, string address, string handle);
+        Message = "Sending push notification (type '{eventName}') to the identity with handle '{handle}'.")]
```

Comment:
I think the push notification handle is considered personal data as well.

Reply:
Changes made.